src/Controller/InfoController.php

```diff
@@ -15,28 +18,21 @@
  * limitations under the License.
  */
 
-namespace App\Controller;
+namespace Surfnet\Tiqr\Controller;
 
 use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 class InfoController extends AbstractController
 {
-    /**
-     * @var LoggerInterface
-     */
-    private $logger;
-
-    public function __construct(LoggerInterface $logger)
+    public function __construct(private readonly LoggerInterface $logger)
     {
-        $this->logger = $logger;
     }
 
-    /**
-     * @Route("/info.html", name="app_info", methods={"GET"})
-     */
-    public function infoAction()
+    #[Route(path: '/info.html', name: 'app_info', methods: ['GET'])]
+    public function info(): Response
     {
         $this->logger->notice('User requested the info.html page');
         return $this->render('default/info.html.twig', []);
```

Comment:
When to use default as a path and when as a namespace?

Reply:
Yes...

Somethimes you use `$this->render('default/info.html.twig', []);` and in other situation `$this->render('@default/info.html.twig', []);` (notice the @) 

I wondered if there's an reason or advantage using one over the other

Reply:
Ah, now I get it. The short answer is no, there is no advantage that I know of. And I'd like to have it uniformly used. Making work of it right away sir!